Demos/FluentUIDemo_iOS/FluentUIDemoTests/NotificationViewTest_SwiftUI.swift

```diff
@@ -67,6 +67,9 @@ class NotificationViewTestSwiftUI: BaseTest {
         let notificationView: XCUIElement = app.otherElements.containing(NSPredicate(format: "identifier MATCHES %@", "Notification View.*")).element(boundBy: 7)
         let actionButton: XCUIElement = app.buttons["Undo"].firstMatch
 
+        // Switches may be offscreen, so scroll first
+        app.swipeUp()
```

Comment:
interesting.... I believe xcuitests used to auto-scroll because they just search through the accessibility tree. but maybe this is the change you were alluding to

Reply:
I think it might be SwiftUI-specific, actually, but I'm not 100% sure.